src/renderers/dom/shared/ReactDOMInjection.js

```diff
@@ -6,7 +6,7 @@
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
  *
- * @providesModule ReactDefaultInjection
+ * @providesModule ReactDOMInjection
```

Comment:
@iamdustan This file might be of interest to you. It is the remainder of the injection system. At least in the first version of Fiber, but we can probably get rid of these completely at some point.